src/test/java/org/glassfish/logging/annotation/LogMessagesResourceBundleGeneratorTest.java

```diff
@@ -99,7 +100,7 @@ public void testNonStandardResourceBundleName() {
         String output = executeCompiler(f1);
         assertTrue(output.contains("annotated by @LogMessagesResourceBundle does not end with 'LogMessages'"));
         File[] resourceBundles = getResourceBundles();
-        assertEquals(resourceBundles.length,0);        
+        assertEquals(0, resourceBundles.length);
```

Comment:
Maybe also a description would be useful, like this?
```suggestion
        assertEquals(0, resourceBundles.length, "resourceBundles should be empty");
```

Reply:
IMO - the suggested description does not help much. If any - it should rather be something like:

_There should be no RB generated for \<case>..._

To describe the intention, and not to decompile `len != 0`.

That's what I think.

Reply:
Even my message helps a bit, to know what went wrong without looking into code. Without it, JUnit will just give you an information that 0 doesn't equal N, and the stacktrace.

What you suggest would be even better, to give some context.

Reply:
I'd like to stay within the scope advertised by commit message and PR title. Can we have those mistakes fixed, and deal with descriptions later?